app/Http/Requests/SignUpRequest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rules;
+
+class SignUpRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => ['required', 'string', 'min: 5', 'max:100'],
+            'email' => ['required', 'string', 'email', 'max:255', 'unique:users'],
+            'password' => ['required', 'string','confirmed', Rules\Password::defaults()],
```

Comment:
```suggestion
            'password' => ['required', 'string','confirmed', Password::defaults()],
```
m suggest `use Illuminate\Validation\Rules\Password;` ở trên cùng lun nhé